### DISCUSSION TITLE
Why do modifiers have to be used directly with blocks

### DISCUSSION THREAD

**Discussion by crazyjaw** [getbem/getbem.github.io#265] [Q&A]
The http://getbem.com/naming/ bit on modifiers says this is "Good":

```
<div class="block block--mod">...</div>
    <div class="block block--size-big
        block--shadow-yes">...</div>
```

Which implies that you must always have the `block` _directly_ along with the `block--mod`, and being a child of a `block` doesn't cut it.

I dont have an understanding of why this isn't acceptable:

```
<div class="block>...</div>
    <div class="block--mod">...</div>
```

**Comment by tadatuta** [comment]
Is [this](https://en.bem.info/methodology/faq/#why-can39t-i-write-block_mod-instead-of-block-block_mod-if-the-modifier-name-already-contains-all-the-block-information) an answer to your question?

**Comment by crazyjaw** [comment]
hmm ive seen that, and it doesn't address it directly (seems to talk about `class="block block--mod"` vs `class="block--mod"`, rather than the nesting bit, which actually makes sense to me)
